File: src/main/java/net/softbell/bsh/iot/controller/rest/v1/IotRestV1.java
package net.softbell.bsh.iot.controller.rest.v1;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import lombok.AllArgsConstructor;
import net.softbell.bsh.iot.component.v1.IotAuthCompV1;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : IoT REST API 컨트롤러 V1
 */
@AllArgsConstructor
@RestController
@RequestMapping("/api/rest/v1/iot")
public class IotRestV1
{
	private final IotAuthCompV1 iotAuthComp;
	
	@GetMapping("/send")
	public String procSend()
	{
		// Field
		
		
		// Finish
		return "Success";
	}
}

File: src/main/kotlin/net/softbell/bsh/iot/controller/rest/v1/IotRestV1.kt
package net.softbell.bsh.iot.controller.rest.v1

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

/**
 * @author : Bell(bell@softbell.net)
 * @description : IoT REST API 컨트롤러 V1
 */
@RestController
@RequestMapping("/api/rest/v1/iot")
class IotRestV1 {
    @GetMapping("/send")
    fun procSend(): String {
        // Init

        // Finish
        return "Success"
    }
}